fcrepo-persistence-api/src/main/java/org/fcrepo/persistence/api/CreateResourceResponse.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Licensed to DuraSpace under one or more contributor license agreements.
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * DuraSpace licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except in
+ * compliance with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.fcrepo.persistence.api;
+
+/**
+ * A class describes the result of creating a resource.
+ * @author dbernstein
+ */
+public class CreateResourceResponse {
```

Comment:
In the Google document you were returning the `FedoraResource` for all the various interactions, so I am wondering what these 3 new responses hold and how they are different from each other and from a GET request response?

Reply:
I was thinking having a "response" object leaves open the door for returning other metadata related to the result - not just the FedoraResource.